pages/docs/concepts/server.md

```diff
@@ -0,0 +1,32 @@
+
+## What is a server?
+A server is a representation of a messaging broker system where connections and communication between a producer and a consumer is established. Unlike traditional API servers which are dependent on request/response, message broker interactions occur back and forth over different channels.
+
+## What is the purpose of servers?
+Servers play an important role as they maintain a relationship between producers and consumers. When designing and setting up an event-driven application, servers are in charge of delivering asynchronous messages from the producer to the consumers through the use of channels. With the integration of different messaging protocols, servers can transmit and exchange messages between clients.
+
+* Clients and Server
+```mermaid
+flowchart TD
+    a[Client Browser] --> b[(server)]
+    b --> a
+    c[Client Mobile] --> b[(server)]
+    b --> c 
+```
+The diagram above describes a bi-directional relationship between `clients` and `server`. When we create an AsyncAPI `server` file, the AsyncAPI server object holds information that includes the physical location of the `server`.
```

Comment:
Hie @smoya could you please kindly review

Reply:
@smoya pingy pongo 😄

Reply:
![Who dares disturb my slumber Aladding meme](https://i.kym-cdn.com/photos/images/newsfeed/001/293/178/69b.jpg)

Kidding 🤣 , I was on vacation until yesterday and I didn't fully catch-up with notifications. 
Ok, I did a review of the last added sentences, which I think is what has been added since last review @derberg did.

As an extra thing, I would like to suggest we use Grammarly before merging so we can catch possible typos 🔍 ✍️ .

Reply:
Yes @smoya, her work still needs an editorial review. I have not reviewed this PR because I want her to get technical feedback first. 

That said, @thulieblack, you should be running this thro Grammarly now, not wait till the final draft. You always want to get into the habit of using Grammarly from the very start on your first draft. 😄

Reply:
lol believe me I use Grammarly on everything. Maybe it took a holiday while I was busy writing the draft😅